### ISSUE TITLE
Opening chunked array files fails

### ISSUE THREAD

**Issue by JovanVeljanoski** [vaexio/vaex#585]
An issue to track the support for chunked arrays (primarily for arrow files). 

@xdssio if you can provide some sample data for testing, that would be great.

**Comment by xdssio** [comment]
[File example](s3://xdss-tmp/examples/simple_test_with_batch.arrow)

**Comment by byaminov** [comment]
Here's another reproducible case. Try to read a small CSV with `pyarrow` and you get a `ChunkedArray` inside. The following script could be copied as a unit test:

```
import vaex
from pyarrow import csv

csv_path = 'vaex/tests/data/small3.csv'
table = csv.read_csv(csv_path)
df = vaex.from_arrow_table(table)
# this works
assert len(df) == 3
# this fails: there's only [5, 7]
assert df.columns['x'] == [5, 7, 9]
# this fails with ValueError: could not broadcast input array from shape (2) into shape (3)
assert (df.x.values == [5, 7, 9]).all()
```

As this happens not only for big tables, I'd re-categorize this issue to a "bug".

**Comment by maartenbreddels** [comment]
I think this should be solved once #517 is merged since we then fully support arrow.

**Comment by JovanVeljanoski** [comment]
This is now fixed. Thanks everyone.
Please reopen if you encounter new issues.
